app/controllers/watched_talks_controller.rb

```diff
@@ -1,4 +1,6 @@
 class WatchedTalksController < ApplicationController
+  include WatchedTalks
+
   def index
     @talks = Current.user.watched_talks
       .includes(talk: [:speakers, {event: :organisation}, {child_talks: :speakers}])
```

Comment:
not really related to this PR but while we are on it I think it could be a good time to improve that 

here we have an @talk instance variable but it is a collection of WatchedTalk. I was tracking a small N+1 on that page and realised that the current code isn't really easy to follow. 

I think we should rely more on the watch_talk and try to avoid having this line 

<% watched_talk = user_watched_talks.find { |wt| wt.talk_id == talk.id } %> in the _card partial